### PR TITLE
tests: Updated include guards to remove leading underscores

### PR DESCRIPTION
**Pull request by altairpearl**
Refering to the issue #2623. Have modified the module tests  include guards . Please review .

### CODE REVIEW

**Comment by authmillenon** [comment]
ACK, when Travis is happy.

File: tests/unittests/common/unittests-constants.h [hunk 1]
```diff
@@ -6,8 +6,8 @@
  * directory for more details.
  */
 
-#ifndef __UNITTESTS_CONSTANTS_H__
-#define __UNITTESTS_CONSTANTS_H_
+#ifndef UNITTESTS_CONSTANTS_H__
```
Comment (haukepetersen) [inline tests/unittests/common/unittests-constants.h]: here is one trailing undersore too much

**Comment by haukepetersen** [comment]
please address the comment above, then I will ack as well.

**Comment by authmillenon** [comment]
Re-ACK, please rebase to current master and squash the commits to one.

**Comment by altairpearl** [comment]
okay. I'm on it.

**Comment by authmillenon** [comment]
Great! Now we only need to wait for Travis.

Thanks for your contribution

**Comment by altairpearl** [comment]
hey ,I have a doubt .Should I create individual pull requests for each such enhancements or can i solve issues for multiple modules and generate a single pull request for all of them at once ? Because I have solved some more issues and wan't to know whether i can add some more commits to this PR .

**Comment by authmillenon** [comment]
Since this PR is ACK'd already you would have to wait for another review. So I would suggest to open a new PR.

**Comment by altairpearl** [comment]
okay got it ! thanks

**Comment by altairpearl** [comment]
ACK Travis is happy now .

**Comment by PeterKietzmann** [comment]
This PR is ACKed and Travis is happy. Go

**Pull request merged**